### PR TITLE
Feature/my address

### PR DESCRIPTION
**Pull request by AnirupPat**

### CODE REVIEW

File: src/app/add-address/add-address.component.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      //Constants.ADDRESS_TYPES[0].value,
+      1, // check Constants.ADDRESS_TYPES for different types of addresses. Only those should be used
+      formData.add1,
+      formData.town_city,
+      formData.county,
+      formData.country,
+      formData.postCode,
+      formData.add2
+    )
+    console.log(this.addressDetails)
```
Comment (reddy-s) [inline src/app/add-address/add-address.component.ts]: We can get rid of this @AnirupPat

File: src/app/address/address.component.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          <p>Add Address</p>
+      </div>
+      <div class="spinner-div">
+          <mat-spinner strokeWidth="4" [diameter]="50" *ngIf="showSpinner"></mat-spinner>
+      </div>
+    </a>
+    <div class="rapido-add-address-item" *ngFor="let address of address">
+      <div class="address-text">
+        <strong> {{ address.full_name }} </strong>
+        <br /><p>{{ address.addr_1 }}
```
Comment (reddy-s) [inline src/app/address/address.component.html]: Formatting of <br/>

Comment (reddy-s) [inline src/app/address/address.component.html]: Formatting of "<br/>" is needed

File: src/app/edit-address/edit-address.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    fixture = TestBed.createComponent(EditAddressComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should not create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  /*it('Mandatory fields during edit Address', async(() => {
```
Comment (reddy-s) [inline src/app/edit-address/edit-address.component.spec.ts]: I will pick this one. Please make a note from your side too, so that we don't forget

Comment (AnirupPat) [reply src/app/edit-address/edit-address.component.spec.ts]: sure

**Pull request merged**